### PR TITLE
Fixes #17828: Inject short hostname as a node property to be able to use it in url/json path

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17828

### CODE REVIEW

File: datasources/build.conf [hunk 1]
```diff
@@ -23,7 +23,7 @@ in <a href="/secure/administration/dataSourceManagement">the dedicated managemen
 # - x.y(.z): plugin major.minor.micro. Micro should be omitted. When omitted, z is assumed to be 0.
 # For the build, we split the information between two properties, rudder branch and plugin version,
 # which must be concaneted with "-" to build the plugin version.
-plugin-branch=1.9-SNAPSHOT
+plugin-branch=1.9-POC-inject-shorthostname
```
Comment (VinceMacBuche) [inline datasources/build.conf:26]: you can remove this ;)

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by fanf** [comment]
OK, merging this PR

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**